aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -3490,8 +3520,8 @@ static std::string get_default_lstsq_driver(c10::optional<c10::string_view> driv
       );
     } else { // else if (input.is_cuda())
       TORCH_CHECK(
-        driver_str == "gels",
-        "torch.linalg.lstsq: `driver` other than `gels` is not supported on CUDA"
+        (driver_str == "gels" || driver_str == "gelsd"),
+        "torch.linalg.lstsq: `driver` other than `gels` or `gelsd` is not supported on CUDA"
```

Comment:
with the if case
`if (driver == "gels" || driver == "gelsd")` i added this so it would pass the assertion.

Reply:
should be gelss.